subsys/net/ip/net_shell.c

```diff
@@ -6096,6 +6237,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(net_commands,
 		  "Print information about IPv6 specific information and "
 		  "configuration.",
 		  cmd_net_ipv6),
+	SHELL_CMD(ip, &net_cmd_ip,
```

Comment:
Let's use `ipv4` command for consistency with IPv6 approach. Perhaps in the future we may want to introduce generic `ip` command to deal with any type of address.

Reply:
Done.